PowerEditor/src/WinControls/ProjectPanel/ProjectPanel.cpp

```diff
@@ -802,6 +802,24 @@ void ProjectPanel::notified(LPNMHDR notification)
 			break;
 		}
 	}
+	else if (notification->code == NM_CUSTOMDRAW && (notification->hwndFrom == _hToolbarMenu))
+	{
+		NMTBCUSTOMDRAW* nmtbcd = reinterpret_cast<NMTBCUSTOMDRAW*>(notification);
+		//if (nmtbcd->nmcd.dwDrawStage == CDDS_PREERASE)
+		{
+			if (NppDarkMode::isEnabled())
+			{
+				FillRect(nmtbcd->nmcd.hdc, &nmtbcd->nmcd.rc, NppDarkMode::getBackgroundBrush());
+				nmtbcd->clrText = NppDarkMode::getTextColor();
+				SetTextColor(nmtbcd->nmcd.hdc, NppDarkMode::getTextColor());
+				SetWindowLongPtr(_hSelf, DWLP_MSGRESULT, CDRF_SKIPDEFAULT);

```

Comment:
This will not work. Because visual style is enabled.
For FolderAsWorkspace [linea 717-718](https://github.com/notepad-plus-plus/notepad-plus-plus/blob/master/PowerEditor/src/WinControls/FileBrowser/fileBrowser.cpp#L717) have no effect because there is no text.
Even with text `SetTextColor` will have no effect.

Reply:
@ozone10 
So do you have any suggestion?

Reply:
I would remove FolderAsWorkspace lines 717-718, as they do nothing.

For this:
You can disable visual style with `SetWindowTheme(_hToolbarMenu, L"", L"");`.

I would probably add it to NppDarkMode.cpp so we don't have to use `#include <Uxtheme.h>`

quickly prototyping, this seems to work, you can adapt it
```
case WM_NOTIFY:
		{
			auto notification = reinterpret_cast<LPNMHDR>(lParam);

			switch (notification->code)
			{
				case NM_CUSTOMDRAW:
				{
					if (NppDarkMode::isEnabled() && notification->hwndFrom == _hToolbarMenu)
					{
						auto nmtbcd = reinterpret_cast<LPNMTBCUSTOMDRAW>(notification);
						FillRect(nmtbcd->nmcd.hdc, &nmtbcd->nmcd.rc, NppDarkMode::getBackgroundBrush());
						nmtbcd->clrText = NppDarkMode::getTextColor();
                                                // highlight color when hover
                                                // same color when hovering above menu 
                                                // RGB(65, 65, 65) should be added to NppDarkMode.cpp
                                                // needed because, visual style is disabled
						nmtbcd->clrHighlightHotTrack = RGB(65, 65, 65); 
						SetWindowLongPtr(_hSelf, DWLP_MSGRESULT, CDRF_NOTIFYSUBITEMDRAW | TBCDRF_HILITEHOTTRACK);
					}
					else
					{
						SetWindowLongPtr(_hSelf, DWLP_MSGRESULT, CDRF_DODEFAULT);
					}
					break;
				}
				default:
				{
					notified(notification);
				}
			}
		}
return TRUE;
```

For function list it should also work

Reply:
@ozone10 
Thank you! Your solution works fine.
Please let me know if you have suggestion about the current state of PR.

Reply:
Maybe when switching back to light mode, enable visual style with this `SetWindowTheme(hwnd, nullptr, nullptr);`

Reply:
> Maybe when switching back to light mode, enable visual style with this SetWindowTheme(hwnd, nullptr, nullptr);

Without doing so, it switches back without problem on my machine... I will take as it is now. We will see if it's really needed on some different configuration.

Reply:
@ozone10 
> I would remove FolderAsWorkspace lines 717-718, as they do nothing.

Line 717 & 718 contains a curly opened brace and a empty line respectively.
Could you point out explicitly the no effect lines please?

Reply:
@ozone10 
Removed : https://github.com/notepad-plus-plus/notepad-plus-plus/commit/5df7c97f98915ed293366d9d521fe5115b2c7c45

Thank you. 

> also if visual style is not enabled, highlight seems different
you can check it when starting in light mode, enable dark mode and then disable without restart

I cannot see the difference :(

Reply:
@donho
start in light mode
![image](https://user-images.githubusercontent.com/55940305/122921065-97f29000-d351-11eb-9ec1-d82f042fcc4c.png)

switch from dark mode
![image](https://user-images.githubusercontent.com/55940305/122921198-beb0c680-d351-11eb-8a22-d15f2483d86c.png)

Reply:
@ozone10 
Thank you for screenshots.
Fixed in https://github.com/notepad-plus-plus/notepad-plus-plus/commit/296e1c695e0e38a4233ab2a70278fcb675419e63